src/client/visualizations/table/table.tsx

```diff
@@ -50,6 +51,7 @@ const ROW_HEIGHT = 30;
 const SPACE_LEFT = 10;
 const SPACE_RIGHT = 10;
 const HIGHLIGHT_BUBBLE_V_OFFSET = -4;
+const MAX_SEGMENT_WIDTH = Number.MAX_SAFE_INTEGER;
```

Comment:
Probably we should set there something like table width.

Reply:
I will change this. You can actually move the column of the screen and then can't grab it again. Great catch!